docs/installation/install_docker.md

```diff
@@ -0,0 +1,35 @@
+# Installing via Docker
+
+## Get Docker
+
+Docker is a containerization platform that allows you to run applications in a sandboxed environment. You can install Docker on your local machine or on a server. For more information, see the [Docker documentation](https://docs.docker.com/).
+
+You can download docker from here: [Get Docker](https://docs.docker.com/get-docker/).
```

Comment:
For mac, Docker desktop was brutal.  I switched to colima, it seems to be better.

Hard to say what docs are necessary for Docker on the whole.

These docs will go in the wiki :-)  Thanks for this doc!